### ISSUE TITLE
Voting Strategy: Update snapshot strategy to support H20

### ISSUE THREAD

**Issue by idiom-bytes** [oceanprotocol/oceandao#270]
We can add another ERC20 to the strategy, like ocean, but it won't be equivalent to Ocean, so this math becomes complicated to solve.

Also, we may not be able to look at H20 balances because the price of the underlying will change daily. So, we should likely look at the underlying lock/depost, to understand voting power.
 
Based on Ops:
- We are not going to add it to strategy this round
- This requirement came in very late, so it would be breaking our dev/shipping cycle
- We would not have enough time to test & verify unless we're doing OT

TLDR; We won't focus on this for R17.
